src/build/mdn-comments.ts

```diff
@@ -27,10 +32,18 @@ function extractSlug(mdnUrl: string): string[] {
     if (!mdnUrl.toLowerCase().startsWith(subdirectory)) {
       continue;
     }
-    return mdnUrl
+    const slugArr = mdnUrl
       .slice(subdirectory.length)
       .replace(/_static/g, "")
-      .split("/");
+      .split("/")
+      .map((part) =>
+        hyphenToCamelCase(part === "Reference" ? "CSSStyleProperties" : part),
+      );
+    if (slugArr[0] === "CSSStyleProperties") {
+      // Drop the second itmem
+      slugArr.splice(1, 1);
```

Comment:
I was confused why this is needed at all, it seems it's because it's now `Web/CSS/Reference/Properties/` as we saw in #2216. Then maybe the subdirectory should be that. (And maybe we should stop lowercasing before doing that)